paper/paper.md

```diff
@@ -67,10 +67,10 @@ to the number of aliases that can be exposed for easy usage.
 
 Creating a registry entry for a scientific container comes down to writing 
 a simple `container.yaml` file with basic metadata and description,
-definition any and all important entrypoints, and the digests to pull.
+the definition of any and all important entrypoints, and the digests to pull.
 As soon as a researcher puts their container in an online registry and adds the
 entry, new versions of the container are automatically discovered by shpc,
-and can be installed by the researcher when he or she chooses.
+and can be installed by the researcher when they choose.
 The user does not need to look in advance for a version if they want the latest provided
 by the registry. Software is easy to search for, and quickly see complete
```

Comment:
```suggestion
by the registry. Software is easy to search for, and with a simple command, the user can quickly see complete
```

Reply:
@danielskatz this is what was implied by the awkward wording - is this better?

Reply:
yes